pull/permission.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021 Palantir Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pull
```

Comment:
I wonder if it would be nice to put this in its own package (instead of `pull`), maybe it would be useful in octo-correct?

Reply:
Given this is a pretty simple type and these two systems don't directly interact, I prefer copying this (or something similar) into other apps that need it rather than introducing a new dependency package or moving it into a common dependency like `go-githubapp` (where I feel it doesn't really fit with any existing functionality.)

Reply:
I was thinking maybe a package in policy-bot, but I think you're right.